sdk/js/src/CacheStore.ts

```diff
@@ -24,17 +30,53 @@ export abstract class CacheStore {
         return this.store?.getItem(storeKey)
     }
 
+    private getConfigStoreKey(user: DVCPopulatedUser) {
+        return user.isAnonymous ? StoreKey.AnonymousConfig : StoreKey.IdentifiedConfig
+    }
+
+    private getUserIdStoreKey(user: DVCPopulatedUser) {
+        return user.isAnonymous ? StoreKey.AnonUserId : StoreKey.IdentifiedConfigUserId
+    }
+
+    private getConfigFetchDateKey(user: DVCPopulatedUser) {
+        return user.isAnonymous ? StoreKey.AnonymousConfigDate : StoreKey.IdentifiedConfigDate
```

Comment:
Rather than duplicating this logic, you could use the getConfigStoreKey to build the key
```suggestion
        return `${getConfigStoreKey(user)}.fetch_date`
```
This would also save us from exporting the user and date keys which shouldn't be accessible anyway